Scripts/RemediationScripts/Remediate-DontUseNonADIdentities.ps1

```diff
@@ -319,6 +333,25 @@ function Remove-AzTSNonADIdentities
     }
 
     $externalAccountsRoleAssignments = @($distinctRoleAssignmentList | Where-Object { $GuestAccountsObjectId -contains $_.ObjectId })
+    }
+    else
+    {
+        Write-Host "Step 2 of 3: Fetching all the role assignments for subscription [$($SubscriptionId)] from given CSV file..."
+        # Connect to Azure Active Directory.
+        try
+        {
+            # Check if Connect-AzureAD session is already active 
+            Get-AzureADUser -ObjectId $currentLoginUserObjectId | Out-Null
```

Comment:
Any reason this validation is required only when using the CSV from dry run?

Reply:
Not required, removed validation from here.